lf-queue/Makefile

```diff
@@ -1,14 +1,21 @@
-CFLAGS =  -Wall -O2 -g -I.
+CC = gcc
```

Comment:
Remove this line since we may use clang as well. Let `make` decide the default compiler i.e., `CC`.